psi4/driver/driver_nbody.py

```diff
@@ -468,7 +468,7 @@ def nbody_gufunc(func, method_string, **kwargs):
         ret_ptype = ret_energy
 
     # Build and set a wavefunction
-    wfn = core.Wavefunction.build(molecule)
+    wfn = core.Wavefunction.build(molecule, 'sto-3g')
```

Comment:
Instead of `sto-3g`, could we use `def2-svp` here? Slightly large, but a much large atom coverage.